pubsub/src/main/java/com/github/twitch4j/pubsub/domain/Prediction.java

```diff
@@ -0,0 +1,27 @@
+package com.github.twitch4j.pubsub.domain;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import lombok.AccessLevel;
+import lombok.Data;
+import lombok.Setter;
+
+import java.time.Instant;
+
+@Data
+@Setter(AccessLevel.NONE)
+@JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class)
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class Prediction {
+    private String id;
+    private String eventId;
+    private String outcomeId;
+    private String channelId;
+    private Integer points;
+    private Instant predictedAt;
+    private Instant updatedAt;
+    private String userId;
+    private PredictionResult result; // can be null
```

Comment:
Can we annotate this with @ Nullable instead of the comment? Lombok should pass that annotation to the getter/setter method.
> lombok.copyableAnnotations = [A list of fully qualified types] (default: empty list)
Lombok will copy any of these annotations from the field to the setter parameter, and to the getter method. Note that lombok ships with a bunch of annotations 'out of the box' which are known to be copyable: All popular nullable/nonnull annotations.